core/src/main/clojure/xtdb/indexer.clj

```diff
@@ -525,8 +527,15 @@
 
                   ^:volatile-mutable indexer-error
 
-                  ^PriorityBlockingQueue awaiters]
+                  ^PriorityBlockingQueue awaiters
 
+                  ^Timer tx-timer
```

Comment:
I'd probably have these as a map, save adding a field every time we want to add a different timer?

Reply:
Yes agree. As this is likely some hot loop I wanted to avoid the extra lookup every time a transaction gets indexed.